### DISCUSSION TITLE
Is there anything else that needs to be merged before 3.7.0 is released?

### DISCUSSION THREAD

**Discussion by ollie-iterators** [jquery/jquery#5239] [Q&A]
### Description ###
I noticed that the 3.7.0 milestone is 100% complete, so I am wondering if there is anything else that needs to be merged before jquery is updated to 3.7.0.

### Link to test case ###
https://github.com/jquery/jquery/milestone/25

**Comment by mgol** [comment]
No, it’s mostly a question of us spending time to prepare a release. It’s likely to happen within the next few weeks.
